src/test/components/__snapshots__/LocalTrack.test.js.snap

```diff
@@ -107,7 +107,7 @@ exports[`timeline/LocalTrack with a network track matches the snapshot of the ne
             class="timelineVerticalIndicatorsLine"
             data-marker-index="10"
             data-testid="vertical-indicator-line"
-            style="--vertical-indicator-color: var(--red-60); left: 66.66666666666667px;"
+            style="--vertical-indicator-color: var(--red-60); left: 200px;"
```

Comment:
(for later) we probably need an aria label with interesting information on these divs

Reply:
Yeah good idea, will file an issue.